packages/webdriverio/src/commands/browser/url.js

```diff
@@ -25,6 +25,8 @@
  * </example>
  *
  * @param {String=} url  the URL to navigate to
```

Comment:
Everywhere this is called path, please adjust the naming here as well ;)

Reply:
Modified